plugin.php

```diff
@@ -19,6 +19,10 @@
 	return;
 }
 
+if ( ! defined( 'ICL_SITEPRESS_VERSION' ) ) {
```

Comment:
@decodekult, we generally do this kind of check later because it depends in which order the plugins are bootstrapped (even though we have some logic in WPML Core to prioritize it). Perhaps you can move it inside `Plugin::init`.

Reply:
In fact, we are doing it again on `Plugin::init`, so I just removed the redundant check here.